mt_with_external_memory/README.eng.md

```diff
@@ -10,9 +10,9 @@ Such models can not only be used in translation, but also other tasks which need
 
 The External Memory Mechanism is this article, is mainly refer to **Neural Turing Machine** \[[1](#reference)\]. It is worth mentioning that Neural Turing Machines is only one of the attempts in the area of simulating Memory Mechanism by neural network. Memory Mechanism has long been widely studied, and recently in the background of deep learning a series of valuable work is developed, like Memory Networks, Differentiable Neural Computer(DNC), etc. In this article only Neural Turing Machine mechanism is discussed and implemented.
```

Comment:
In this article, the external memory mechanism refers to the **Neural Turing Machine** \[[1](#reference)\]. method (which will be explained in detail later)
...attempts of neural network to simulate memory mechanism...
In the context of deep learning in recent years, a series of valuable work has emerged, such as Memory Networks and Differentiable Neural Computers (DNC).
In this article, only...